Purchases/NSError+RCExtensions.m

```diff
@@ -0,0 +1,37 @@
+//
+// Created by Andrés Boedo on 2/24/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+#import "NSError+RCExtensions.h"
+#import "RCPurchasesErrors.h"
+#import "RCBackend.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+@implementation NSError (RCExtensions)
+
+- (BOOL)shouldMarkSyncedKeyPresent {
```

Comment:
I'm still not a fan of this name. first of all, this is true if and only if it the key is present **and** its value is true. I'm open to suggestions